test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "require-jsdoc": 0
```

Comment:
Won't cover extension tests. Is it possible to do this with file glob? E.g. `test-*.js`.

Reply:
Yeah, I was thinking of that. Not sure it's possible to disable specific rules for specific directories that match a pattern.

One way to do so would be to entirely disable linting for extension test directories, but that's bad.

I'll look for a way to do this and check it in separately.

Reply:
Done! Updated commit pushed. Tested locally.